### DISCUSSION TITLE
nvim ignoring .editorconfig

### DISCUSSION THREAD

**Discussion by javihernant** [neovim/neovim#27580] [Q&A]
It is ignoring the .editorconfig file I have in the root of the project.
I'm running nvim v0.9.5. For further info (to see my init.lua) go to my other post [here](https://github.com/neovim/neovim/discussions/27507#discussioncomment-8515378)

**Comment by clason** [comment]
Discussions are not a way of side-stepping the issue template. If you think there's a bug, open an issue **and follow the  guidelines exactly**.

**Reply by javihernant** [reply]
Not my intention. I am more than happy to file an issue. I asked here because I am new at nvim, and not sure if it was a bug or rather that I was doing (configuring) something wrong

**Reply by clason** [reply]
How would _anybody_ be able to answer this question with that little information? If you want to get interactive support, ask on [Matrix](https://matrix.to/#/#neovim:matrix.org).
